workbench-core/base/src/interfaces/SortRequest.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface SortRequest {
```

Comment:
nit: file name should be `sortRequest.ts`

Reply:
fixed in commit